app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -61,19 +61,23 @@ def supports_effective_user?
           true
         end
 
-        def provider_inputs
+        def provider_inputs(provider_input_values = {})
+          tags = provider_input_values&.find { |input| input[:name] == 'tags' }&.fetch(:value) || ''
```

Comment:
I'm just reading this without much context so I might be missing something, but at a glance, If `provider_input_values` is a hash as the default value suggests, then `input` inside `find` is an array of two elements (they key and the value) so accessing `item[:name]` on it will most likely fail.

Reply:
Thanks Adam!